infra/aws/terraform/prow-build-cluster/resources/flux-system/flux-ks-kubecost.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 The Kubernetes Authors.
```

Comment:
Nit: we need to figure out how to generate new manifests with 2024, but let's leave it for a follow up